app/main.py

```diff
@@ -2,0 +1,41 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
```

Comment:
Consider unpacking `command_list` into individual variables for better readability and to adhere to the checklist recommendation.